main/ui.js

```diff
@@ -35,6 +35,9 @@ module.exports = async function (ctx) {
     }
   })
 
+  ui.on('show', () => { ctx.isShowingUI = true })
+  ui.on('hide', () => { ctx.isShowingUI = false })
```

Comment:
Could we use a simpler https://www.electronjs.org/docs/latest/api/browser-window#winisvisible instead?

Reply:
I am not sure if that's the same thing. Quoting from the docs:

> Returns boolean - Whether the window is visible to the user in the foreground of the app.


What if the Station's window is in the background, underneath windows of other applications? What if it's minimised?

Is it worth investing our time to better understand when `win.isVisible()` returns true/false; or is the current solution good enough?